sycl/test/check_device_code/no_offset_error.cpp

```diff
@@ -0,0 +1,21 @@
+// RUN:  %clangxx -fsycl-device-only -fsycl-early-optimizations -fsycl-dead-args-optimization -D__SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__ -Xclang -verify -Xclang -verify-ignore-unexpected=note -S -emit-llvm -o - %s
```

Comment:
```suggestion
// RUN:  %clangxx -fsycl-device-only -Xclang -verify -Xclang -verify-ignore-unexpected=note -emit-llvm -o - %s
```

Reply:
You are right, in that case these flags are extra. I'll remove it.